README.md

```diff
@@ -55,38 +52,58 @@ Or the Matrix server (#emulatorjs:matrix.emulatorjs.org):
 
 <br>
 
-
 ### Issues
 
 *If something doesn't work, please consider opening an* ***[Issue]*** <br>
 *with as many details as possible, as well as the console log.*
 
 <br>
 
+### 3rd Party Projects
+
+EmulatorJS itself is built to be a plugin, rather than an entire website. This is why there is no docker container of this project. However, there are several projects you can use that use EmulatorJS!
+
+Looking for projects that integrate EmulatorJS? Check out https://emulatorjs.org/docs/3rd-party
+
+<br>
+
 ### Versioning
+
 There are three different version names that you need to be aware of:
+
 1. **stable** - This will be the most stable version of the emulator both code and cores will be tested before release. It will be updated every time a new version is released on GitHub. This is the default version on the Demo.
 2. **latest** - This will contain the latest code but use the stable cores. This will be updated every time the *main* branch is updated.
 3. **nightly** - This will contain the latest code and the latest cores. The cores will be updated every day, so this is considered alpha.
 
-### CDN
-There is a new CDN that you can use to get any version of the emulator. The CDN is `https://cdn.emulatorjs.org/`. You can get the stable, latest, nightly, and any other main version by setting your `EJS_pathtodata` to `https://cdn.emulatorjs.org/<version>/data/`.
-
-### Extensions
+<br>
 
- **[GameLibrary]**
+### CDN
 
-   *A library overview for your **ROM** folder.*
+EmulatorJS provides a CDN at `https://cdn.emulatorjs.org/` where you can access any version of the emulator. To use it, set your `EJS_pathtodata` to `https://cdn.emulatorjs.org/<version>/data/`, replacing <version> with stable, latest, nightly, or any other main release you need.
```

Comment:
Please also note that you should also change the path to loader.js. It works either way though one isn't technically officially supported